test/components/submission/update-review-state.spec.js

```diff
@@ -59,55 +44,45 @@ describe('SubmissionUpdateReviewState', () => {
   });
 
   describe('review state selection', () => {
-    it('sets the selection to the current review state', () => {
+    it('sets the selection to the current review state', async () => {
       testData.extendedSubmissions.createPast(1, { reviewState: 'hasIssues' });
-      const radio = mountComponent().first('input[value="hasIssues"]');
-      radio.element.checked.should.be.true();
+      const modal = mountComponent();
+      await setProps(modal, { state: true });
+      modal.first('input[value="hasIssues"]').element.checked.should.be.true();
     });
 
-    it('sets the selection to approved if current review state is null', () => {
+    it('sets selection to approved if current review state is null', async () => {
       testData.extendedSubmissions.createPast(1, { reviewState: null });
-      const radio = mountComponent().first('input[value="approved"]');
-      radio.element.checked.should.be.true();
+      const modal = mountComponent();
+      await setProps(modal, { state: true });
+      modal.first('input[value="approved"]').element.checked.should.be.true();
     });
 
-    it('sets the selection to approved if current review state is edited', () => {
+    it('sets selection to approved if current review state is edited', async () => {
       testData.extendedSubmissions.createPast(1, { reviewState: 'edited' });
-      const radio = mountComponent().first('input[value="approved"]');
-      radio.element.checked.should.be.true();
+      const modal = mountComponent();
+      await setProps(modal, { state: true });
+      modal.first('input[value="approved"]').element.checked.should.be.true();
     });
   });
 
-  it('focuses the review state radio', () => {
+  it('focuses the review state radio', async () => {
     testData.extendedSubmissions.createPast(1, { reviewState: 'hasIssues' });
     const modal = mountComponent({ attachToDocument: true });
+    await setProps(modal, { state: true });
     modal.first('input[value="hasIssues"]').should.be.focused();
   });
 
-  describe('resetting the form', () => {
-    it('resets the form after the modal is hidden', async () => {
-      testData.extendedSubmissions.createPast(1, { reviewState: 'hasIssues' });
-      const modal = mountComponent();
-      await trigger.check(modal, 'input[value="rejected"]');
-      await trigger.input(modal, 'textarea', 'Some notes');
-      modal.setProps({ state: false });
-      await modal.vm.$nextTick();
-      modal.setProps({ state: true });
-      await modal.vm.$nextTick();
-      modal.first('input[value="hasIssues"]').element.checked.should.be.true();
-      modal.first('textarea').element.value.should.equal('');
-    });
-
-    it('resets review state to approved if current state is null', async () => {
```

Comment:
The review state selection is now set in only one place, not two. That means that this is no longer a special case and is covered by the tests above. I've removed this test.